frontend/src/screens/actions/list.vue

```diff
@@ -74,14 +74,20 @@
         @onDelete="onDelete"
         @onQuickView="onQuickView"
         @onSort=onSort>
-      </simple-table>
-      <pagination-table-component
-        :dataLength="actions.length"
-        :pagination="pagination"
-        @goNext="goNext"
-        @goPrevious="goPrevious"
-        @goToPage="goToPage">
-      </pagination-table-component>
+    </simple-table>
+    <pagination-table-component v-if="actions.length"
+      :dataLength="actions.length"
+      :pagination="pagination"
+      @goNext="goNext"
+      @goPrevious="goPrevious"
+      @goToPage="goToPage">
+    </pagination-table-component>
+    <div v-if="actions.length == 0">
```

Comment:
No es necesario agregar el `div` el `v-if` puede ir directo en `missing-data-empty-state` y no hace falta comparar contra el 0.
```suggestion
    <missing-data-empty-state v-if="actions.length">
```